src/features/fee-nonce-drawers/transaction-settings-drawer.tsx

```diff
@@ -42,11 +43,15 @@ const EditNonce = () => {
 };
 
 const Messaging = () => {
+  const url = '';
   return (
-    <Caption>
-      Apply a higher fee to help your transaction confirm quickly, especially when the network is
-      congested by other transactions.
-    </Caption>
+    <>
+      <Caption>
+        Apply a higher fee to help your transaction confirm quickly, especially when the network is
+        congested by other transactions.
+      </Caption>
+      <Link onClick={() => window.open(url, '_blank')}>Learn more...</Link>
```

Comment:
we dont do this currently, but we should use a method like this:
```ts
const openInNewTab = (url: string) => {
    const newWindow = window.open(url, '_blank', 'noopener,noreferrer')
    if (newWindow) newWindow.opener = null
}
```

[See this for context](https://stackoverflow.com/a/63627759)

Reply:
Beyond scope of this change, but preferably validating it's a URL being passed in is legit to, see external-links.ts in desktop wallet